packages/scandipwa/src/store/Config/Config.reducer.js

```diff
@@ -23,13 +23,22 @@ export const filterStoreConfig = (config) => Object.entries(config).reduce(
     {}
 );
 
+/** @namespace Store/Config/Reducer/filterAvailableCurrencies */
+export const filterAvailableCurrencies = ({ available_currencies_data: availableCurrencies },
+    { base_currency_code: base, exchange_rates: rates }) => ({
```

Comment:
Please place this structure to separate lines.
And don't forget to assign default values in case if some data structures are undefined at some point.

Reply:
When you are working with the Redux, please note that it may have undefined, empty values on the first render, and then it's populated with data.
In order to improve the performance of the app, we may cache the data in the localStorage.
But the new customers will have the empty localStorage.
So you should test this case too when you are working with the Redux.